frontend/src/stories/ReviewsTable.stories.js

```diff
@@ -0,0 +1,9 @@
+import React from "react";
```

Comment:
We dont need a new story. We need the existing stories to draw the correct things.

Run the storybook and look at these urls:
http://localhost:6006/?path=/story/tilde-carddetailsmodal-reviews--loading
http://localhost:6006/?path=/story/tilde-carddetailsmodal-reviews--primary

These need to draw the correct thing

Reply:
frontend/src/stories/ReviewsTable.stories.js should be deleted

Reply:
The http://localhost:6006/?path=/story/tilde-carddetailsmodal-reviews--primary now draws the correct things. I have not edited the http://localhost:6006/?path=/story/tilde-carddetailsmodal-reviews--loading